pkg/apis/ceph.rook.io/v1/validate_clusters.go

```diff
@@ -62,6 +62,10 @@ func (c *CephCluster) ValidateDelete() error {
 }
 
 func validateUpdatedCephCluster(updatedCephCluster *CephCluster, found *CephCluster) error {
+	if updatedCephCluster.Spec.Mon.Count > 0 && updatedCephCluster.Spec.Mon.Count%2 == 0 {
```

Comment:
What about making `Count` an `uint` to avoid the `> 0 check?

Reply:
It's skipping over the case where the mon count is 0 (external cluster)